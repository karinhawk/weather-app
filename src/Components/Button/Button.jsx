import "./Button.scss"

const Button = ({fetchData, localWeather}) => {
  return (
    <div>
        <button className='button' onClick={() => fetchData(localWeather)}>Local Weather</button>
    </div>
  )
}

export default Button