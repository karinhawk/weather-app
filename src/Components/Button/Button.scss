.button {
    height: 5vh;
    padding: 0.5rem;
    text-align: center;
    border: none;
    border-radius: 0.5rem;
    
    &:hover {
        cursor: pointer;
    }
}