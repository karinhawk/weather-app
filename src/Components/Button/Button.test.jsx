import Button from "./Button.jsx"
import { render } from "@testing-library/react";

it("Should render the button", () => {
    const {asFragment} = render(<Button/>)
    expect(asFragment(<Button />)).toMatchSnapshot()
  });