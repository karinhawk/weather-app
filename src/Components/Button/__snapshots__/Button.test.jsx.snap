// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the button 1`] = `
<DocumentFragment>
  <div>
    <button
      class="button"
    >
      Local Weather
    </button>
  </div>
</DocumentFragment>
`;
